package-lock.json

```diff
@@ -19607,9 +19607,9 @@
       "integrity": "sha512-qIy6tLx8rtybEsIOAlrM4J/85s2q2nPkDqj/Rx46VakBZ0LwtFhXIVub96LXHczQX0vaqmAueDqNPXtbSXSaYQ=="
     },
     "@openzeppelin/contracts-v4": {
-      "version": "npm:@openzeppelin/contracts@4.4.2",
-      "resolved": "https://registry.npmjs.org/@openzeppelin/contracts/-/contracts-4.4.2.tgz",
-      "integrity": "sha512-NyJV7sJgoGYqbtNUWgzzOGW4T6rR19FmX1IJgXGdapGPWsuMelGJn9h03nos0iqfforCbCB0iYIR0MtIuIFLLw=="
+      "version": "npm:@openzeppelin/contracts@4.3.1",
```

Comment:
`package-lock.json` has changed but there aren't any changes in `package.json`. Did you intend to commit this file or was it a mistake?